client/state/http/actions.js

```diff
@@ -26,7 +26,6 @@ export const http = ( {
 	url,
 	method,
 	headers,
-	queryParams,
```

Comment:
Hmm, I think the mistake here was that it should have been included in the generated action return object, since it's later expected in the HTTP handler:

https://github.com/Automattic/wp-calypso/blob/a40fa964c9115e056041a849973f4b61b76c1ae7/client/state/http/index.js#L39

cc @yurynix @dmsnell (#16036, d2dff9535b67f2125103ea52866ff350bd4d7dcf).

Reply:
Makes sense - added it back, solved the issue, and updated the docs. Yuri, Dennis: would you mind confirming that's what you intended and provide testing instructions?

Reply:
The tests @nosolosw added looks good.
There isn't anything that uses `queryParams` for the generic http handler afaik, so the only tests are unit tests.